Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/CreateReservation/CreateReservationRequestValidator.cs

```diff
@@ -0,0 +1,63 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests.Reservation;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Application.Utilities;
+using Looms.PoS.Domain.Interfaces;
+
+namespace Looms.PoS.Application.Features.Reservation.Commands.CreateReservation;
+
+public class CreateReservationRequestValidator : AbstractValidator<CreateReservationRequest>
+{
+    private readonly IServicesRepository _servicesRepository;
+    private readonly IReservationsRepository _reservationsRepository;
+
+    public CreateReservationRequestValidator(IServicesRepository servicesRepository, IReservationsRepository reservationsRepository)
```

Comment:
There is no validation for service availability per employee. For example, employee can make multiple reservations in the same time span for the same employee. Shouldn't be possible.

Reply:
It's not nullable in class diagram, only in api contract. I believe if it's null, then current user should be assigned. We can just make it required..

Reply:
made employee id required and added validation for it. But the validation is not perfect it just checks if it starts at the same time not overlaps. I can make it better but Titas before said its fine to leave it as this